src/app/components/timesheet/employeeCard.tsx

```diff
@@ -0,0 +1,48 @@
+import React, {memo} from 'react';
+import {StyleSheet, View} from 'react-native';
+
+import Typography from '../typography';
+
+import {RightArrow} from '../../constant/icons';
+
+import {flexStyles} from '../../../styles';
+
+type Props = {
+  details: {
+    employee_id: string;
+    name: string;
+    email: string;
+  };
```

Comment:
Instead of getting the details object as a prop, we can pass each item as a separate prop which avoids re-rendering